vsm

```diff
@@ -547,6 +843,8 @@ if __name__ == "__main__":
                 logger.e(err)
                 exit(1)
 
+        config_tree = TreeNode(NODE_ROOT, None)
```

Comment:
I see this variable only being used in one place to add a child, what is exactly its function?